bot/persistence/database_connector.py

```diff
@@ -49,11 +46,8 @@ def add_config_property(self, key: str, val: str):
                 is needed to operate on the db. This also ensures that the database exists and works, contains the table
                 prior to calling this method.
         """
-        if self._db_file is None:
-            raise Error(
-                "Method __init__(filename, init_script=None) has not yet been called. Database filename is not set.")
         with DatabaseManager(self._db_file) as db_manager:
-            db_manager.execute(INSERT_PROPERTY_QUERY, (key, val))
+            db_manager.execute(queries.QUERY_INSERT_PROPERTY, (key, val))
```

Comment:
Because the more features we implement, the more queries we would have to manually import which I thought would be kinda tideous. That's why I imported the whole module.

Reply:
I personally prefer explicit Imports, especially since it makes the code where the constants are used shorter and easier to read.